devtools/gradle/gradle-application-plugin/build.gradle

```diff
@@ -7,6 +7,7 @@ dependencies {
     implementation "io.quarkus:quarkus-devtools-common:${version}"
     implementation "io.quarkus:quarkus-core-deployment:${version}"
     implementation "io.quarkus:quarkus-bootstrap-gradle-resolver:${version}"
+    implementation "io.smallrye.config:smallrye-config-source-yaml:${smallrye_config_version}"
```

Comment:
Looks like we should be enforcing the `quarkus-bom`, that's not for this PR though.